### ISSUE TITLE
statement + recursive abstraction constructive critism

### ISSUE THREAD

**Issue by ghost** [helenchan01/semicolons#2]
## statement
Speaking from personal experience, stating more rules that your algorithm must follow can help ease programming. for example try to answer: what will your program do to determine the boolean? What rules must the explorer follow (ex. turns, legal movement)?

## recursive abstraction
I concur with Mr. Holmes' reasoning that going by junctions isn't the best way to approach this problem. Focusing on stepping stones is definitely the way to go! I have also found that thinking of the recursive case as "n-1" problems isn't particularly helpful when it comes to backtracking. Try draw it out, that's what helped me and Jack!

**Comment by helenchan01** [comment]
I completely agree with you. We thought about it after looking at Holmes suggestion and realized our initial methods would just complicate things. Our recursive abstraction is now updated to use stepping stones. Thanks for sharing your insight!
